docs-mslearn/toolkit/changelog.md

```diff
@@ -30,12 +30,16 @@ The following section lists features and enhancements that are currently in deve
 
 ### Documentation improvements
 
+<br><a name="latest"></a>
+
+## v13
+
 - **Enhanced** [Configure scopes documentation](hubs/configure-scopes.md) to explicitly clarify that FinOps hubs support:
```

Comment:
Format this like all the others. Add the FinOps hubs header, indicate the type of change this is, and add the link at the end of the release to compare to the previous release.